### ISSUE TITLE
new seq2seq libraries does not provide the result legacy seq2seq provided.

### ISSUE THREAD

**Issue by PratsBhatt** [tensorflow/nmt#42]
I am using custom database.
First Step:
I was using the old seq2seq tutorial in tensorflow-gpu 1.0.0v. 
The perplexity used to converge to a minimum of 2.1 along with evaluation perplexity of each buckets in an around that value. 

Second Step:
When I upgraded to the new version tensorflow-gpu1.2.0v and used this tutorial to train, the perplexity shows to be 400 and never goes down. The reason of trying the new api was to later do the schedule sampling. But even without that and using the tutorial as it is the results are very bad.

Reason (Maybe):
Is it because the perplexity before printing out is multiplied by the batch size and that increases the value? If so I read the comment on the loss function in the tutorial but did not understand it completely. It would be really nice that I can get a hypothetical answer to this question on what would be happening. 

Great tutorial!
Thank you in advance, appreciate your help and guidance.

**Comment by lmthang** [comment]
Hi, have managed to answered this question? There's only one correct way to compute perplexity. Close for now.
